src/scripts/app.ts

```diff
@@ -118,7 +119,6 @@ export class ComfyApp {
   vueAppReady: boolean
   api: ComfyApi
   ui: ComfyUI
-  extensions: ComfyExtension[]
```

Comment:
`extensions` is stored in a single place now, as there is no extension registered before app.setup after https://github.com/Comfy-Org/ComfyUI_frontend/pull/2047